### PR TITLE
Added output.tf.example to the main terraform root folder

### PR DESCRIPTION
**Pull request by atighineanu**
## What does this PR change?
This PR:
 - allows user to format their own tf.output (by using output.tf.example as source)
 - exposes IPs in the libvirt.host backend module

### CODE REVIEW

File: backend_modules/libvirt/host/main.tf [hunk 1]
```diff
@@ -240,5 +240,8 @@ output "configuration" {
     ids       = libvirt_domain.domain[*].id
     hostnames = [for value_used in libvirt_domain.domain : "${value_used.name}.${var.base_configuration["domain"]}"]
     macaddrs  = [for value_used in libvirt_domain.domain : value_used.network_interface[0].mac if length(value_used.network_interface) > 0]
+    
+    //displays IPs of libvirt domains; use "value_used.network_interface[0].addresses[0]" if you want just IPv4 to be displayed
```
Comment (Bischoff) [inline backend_modules/libvirt/host/main.tf:244]: I would rather remove blank line and comment - this is by no way different than what preceeds, so itt's not useful to comment

File: backend_modules/libvirt/host/main.tf [hunk 2]
```diff
@@ -240,5 +240,8 @@ output "configuration" {
...
+    ips       = [for value_used in libvirt_domain.domain : value_used.network_interface[0].addresses if length(value_used.network_interface) > 0]
```
Comment (Bischoff) [inline backend_modules/libvirt/host/main.tf:245]: just add `[0]` to the end as we mostly use IPv4s to connect to the VMs

File: output.tf.example [hunk 3]
```diff
@@ -0,0 +1,15 @@
+output "machines" {  # you can use any title you want instead of "machines"
```
Comment (Bischoff) [inline output.tf.example:1]: I would suggest to not do this as a snippet. Rather document it in plain English in `README_ADVANCED.md`

**Review by Bischoff — CHANGES_REQUESTED** [review]
nice improvements

see suggestions

File: backend_modules/libvirt/host/main.tf [hunk 4]
```diff
@@ -240,5 +240,8 @@ output "configuration" {
     ids       = libvirt_domain.domain[*].id
     hostnames = [for value_used in libvirt_domain.domain : "${value_used.name}.${var.base_configuration["domain"]}"]
     macaddrs  = [for value_used in libvirt_domain.domain : value_used.network_interface[0].mac if length(value_used.network_interface) > 0]
+
```
Comment (rjmateus) [inline backend_modules/libvirt/host/main.tf:243]: This empty line needs to be removed or the PR checks will not pass. 
Comment can also be removed.

**Review by rjmateus — COMMENTED** [review]
Thanks for your contribution. Just some small note and I agree with Eric's review.

**Comment by rjmateus** [comment]
@atighineanu the PR's title and description don't match the code changes. This is only exposing the IPs on libvirt host module

File: backend_modules/gcp/host/variables.tf [hunk 5]
```diff
@@ -0,0 +1,52 @@
+variable "project_name" {
```
Comment (rjmateus) [inline backend_modules/gcp/host/variables.tf:1]: This file is normally a symbolic link to the host variables file inside the null provider. You can check the other supported providers.
The idea behind it is that sumaform users can use any provider without the need to change several properties in the configuration. 
Also, the module inside `modules/host` can call the same API and don't care about what underlying provider are in use.

**Comment by srbarrios** [comment]
This looks a deprecated PR that can be closed.
@atighineanu would be fine to close it?

**Comment by atighineanu** [comment]
> This looks a deprecated PR that can be closed.
> 
> @atighineanu would be fine to close it?

Sure. There are two years since I have lost contact to sumaform.

**Comment by cbbayburt** [comment]
Closing as obsolete
